src/thundra-wrapper.js

```diff
@@ -100,19 +103,75 @@ class ThundraWrapper {
     async report(error, result, callback) {
         if (!this.reported) {
             this.reported = true;
-            const afterInvocationData = {
+
+            let afterInvocationData = {
                 error: error,
                 response: result
             };
+
+            const skipParseResponse = process.
+                env.thundra_lambda_skip_parse_response === 'true' ? true : false;
+
+            if (skipParseResponse || this.isErrorResponse(result)) {
+                afterInvocationData = {
+                    error: new Error("Lambda returned with error response."),
```

Comment:
We are checking HTTP error codes (4xx and 5xx) therefore we can use specific error type for this case such as `HTTPError`. So user can detect it in the UI. 

Also it would be good to have status code in the error message.

Reply:
👍